src/components/AvatarWithImagePicker.js

```diff
@@ -91,137 +89,141 @@ const propTypes = {
     /** File name of the avatar */
     originalFileName: PropTypes.string,
 
-    ...withLocalizePropTypes,
     ...withNavigationFocusPropTypes,
 };
 
 const defaultProps = {
     source: '',
-    onImageSelected: () => {},
-    onImageRemoved: () => {},
+    onImageSelected: () => { },
+    onImageRemoved: () => { },
     style: [],
-    DefaultAvatar: () => {},
+    DefaultAvatar: () => { },
     isUsingDefaultAvatar: false,
     isUploading: false,
     size: CONST.AVATAR_SIZE.DEFAULT,
     fallbackIcon: Expensicons.FallbackAvatar,
     type: CONST.ICON_TYPE_AVATAR,
     editorMaskImage: undefined,
     errorRowStyles: [],
-    onErrorClose: () => {},
+    onErrorClose: () => { },
     pendingAction: null,
     errors: null,
     headerTitle: '',
     previewSource: '',
     originalFileName: '',
 };
 
-class AvatarWithImagePicker extends React.Component {
-    constructor(props) {
-        super(props);
-        this.animation = new SpinningIndicatorAnimation();
-        this.setError = this.setError.bind(this);
-        this.isValidSize = this.isValidSize.bind(this);
-        this.showAvatarCropModal = this.showAvatarCropModal.bind(this);
-        this.hideAvatarCropModal = this.hideAvatarCropModal.bind(this);
-        this.state = {
-            isMenuVisible: false,
-            validationError: null,
-            phraseParam: {},
-            isAvatarCropModalOpen: false,
-            imageName: '',
-            imageUri: '',
-            imageType: '',
-        };
-        this.anchorRef = React.createRef();
-    }
-
-    componentDidMount() {
-        if (!this.props.isUploading) {
+function AvatarWithImagePicker({
+    isFocused,
+    DefaultAvatar,
+    style,
+    pendingAction,
+    errors,
+    errorRowStyles,
+    onErrorClose,
+    source,
+    fallbackIcon,
+    size,
+    type,
+    headerTitle,
+    previewSource,
+    originalFileName,
+    isUsingDefaultAvatar,
+    onImageRemoved,
+    anchorPosition,
+    anchorAlignment,
+    onImageSelected,
+    editorMaskImage,
+}) {
+    const [isMenuVisible, setIsMenuVisible] = useState(false);
+    const [errorData, setErrorData] = useState({
+        validationError: null,
+        phraseParam: {},
+    });
+    const [isAvatarCropModalOpen, setIsAvatarCropModalOpen] = useState(false);
+    const [imageData, setImageData] = useState({
+        uri: '',
+        name: '',
+        type: ''
+    });
+    const anchorRef = useRef();
+    const { translate } = useLocalize();
```

Comment:
```suggestion
    const {translate} = useLocalize();
```

And same with all other occurrences of space before and after brackets